google-tests/soltrace_example_tests/st_powertower_sim_run_opt_test.cpp

```diff
@@ -0,0 +1,133 @@
+#include <gtest/gtest.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <iostream>
+#include <iomanip>
+#include <fstream>
+#include <sstream>
+#include <cmath>
+
+#include <string>
+#include <cstring>
+#include <vector>
+#include <exception>
+#include <cstdlib>
+
+#include "strace.h"
+#include "stapi.h"
+#include "split_csv.h"
+
+using namespace std;
+
+TEST(st_powertower_sim_run_opt_test, BasicAssertions)
+{
+	cout << "Project Directory: " << PROJECT_DIR << endl;
+	string project_dir = PROJECT_DIR;
+	
+	cout << "Current working directory: " << project_dir << endl;
+
+	string file_path = "/Power-tower-surround_singlefacet.stinput";
+	string csv_path = "/powertower_example_opt_raydata.csv";
+	
+	string input_file_path = project_dir + file_path;
+	string ground_csv_path = project_dir + csv_path;
+
+	cout << "File path: " << project_dir << endl;
```

Comment:
can you clean up the pathing? Remove the cout lines and just directly initialize the paths. For example
```suggestion
	string input_file_path = PROJECT_DIR + "/Power-tower-surround_singlefacet.stinput"
```

Reply:
Removed output lines and directly instantiate paths.